src/main/scala/nl.knaw.dans.easy.validatebag/rules/ProfileVersion1.scala

```diff
@@ -35,19 +35,19 @@ object ProfileVersion1 {
 
     // bag-info.txt
     NumberedRule("1.2.1", containsFile(Paths.get("bag-info.txt"))),
-    NumberedRule("1.2.2", bagInfoContainsExactlyOneOf("BagIt-Profile-Version"), dependsOn = Some("1.2.1")),
-    NumberedRule("1.2.2", bagInfoElementIfExistsHasValue("BagIt-Profile-Version", versionNumber.toString), dependsOn = Some("1.2.2")),
-    NumberedRule("1.2.3", bagInfoContainsExactlyOneOf("BagIt-Profile-URI"), dependsOn = Some("1.2.1")),
-    NumberedRule("1.2.3", bagInfoElementIfExistsHasValue("BagIt-Profile-URI", versionUri), dependsOn = Some("1.2.3")),
-    NumberedRule("1.2.4", bagInfoContainsExactlyOneOf("Created"), dependsOn = Some("1.2.1")),
-    NumberedRule("1.2.4", bagInfoCreatedElementIsIso8601Date, dependsOn = Some("1.2.4")),
+    NumberedRule("1.2.2(a)", bagInfoContainsAtMostOneOf("BagIt-Profile-Version"), dependsOn = Some("1.2.1")),
```

Comment:
@vesaakerman: why have you changed the rule?

Reply:
It was a copy-and-paste error...